### PR TITLE
Landing page update

### PR DESCRIPTION
**Pull request by JoannaNitek**
+ update features section
+ changing in team section
+ update initiatives
+ add use case form

### CODE REVIEW

File: ec0lint/src/index.css [hunk 1]
```diff
@@ -549,6 +549,30 @@ h6 {
   margin-right: auto;
 }
 
+.form {
+  /* display: flex;
```
Comment (malwinq) [inline ec0lint/src/index.css]: let's not leave comments

File: ec0lint/src/pages/MainPage.tsx [hunk 2]
```diff
@@ -35,8 +35,6 @@ function MainPage() {
           </a>
         </div>
         <SupportSection/>
-        <h4>Used by</h4>
```
Comment (malwinq) [inline ec0lint/src/pages/MainPage.tsx:38]: do we want to remove it?

File: ec0lint/src/sections/InitiativesSection.tsx [hunk 3]
```diff
@@ -12,11 +12,15 @@ export default function InitiativesSection() {
   };
 
   const onArrowClickLeft = () => {
-    setCardDisplayed(0);
+    var x = cardDisplayed;
```
Comment (malwinq) [inline ec0lint/src/sections/InitiativesSection.tsx]: please use let instead of var

File: ec0lint/src/sections/ModalForm.tsx [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+                    />
+                </label>
+                <button className="blueSectionButton">Submit</button>
+            </form>
+        </div>
+
+    )
+
+}
+{/* <div className="modal"></div>
```
Comment (malwinq) [inline ec0lint/src/sections/ModalForm.tsx]: please delete the comment

File: ec0lint/src/text.tsx [hunk 5]
```diff
@@ -1120,15 +1384,46 @@ export const initiativesSection = {
...
+            image: <img src={xtremejs}></img>
+        },
+        {
+            title: "Krakowskie Dni Informatyki",
+            text: "We presented a speech during Krakowskie Dni Informatyki (1 000+ attendees) in December 2022.",
+            link: <span>More information <a className="navLinkPurple" target="_blank" href="https://kdi.org.pl/" rel="noreferrer">here</a>.</span>,
+            image: <img src={kdi}></img>
+        },
+        {
+            title: "Goldman Sachs",
```
Comment (malwinq) [inline ec0lint/src/text.tsx]: please delete goldman for now - I'll discuss it with Kasia

**Comment by malwinq** [comment]
can you please add screenshots of changes? we can also review it together on our weekly :)

**Comment by JoannaNitek** [comment]
usecaseform
usecase
features
reactindia
wearedev
ing_zmiany
reactberlin
xtreme
conf42
kdi
initiatives2
Initiatives1
team

**Comment by malwinq** [comment]
overall looks great!

few comments:
* let's change rules listed in futures page to be ordered alphabetically 
* we can move Ola to Contributors section
* keep the blue section in initiatives with const height (min height), because at the last page there's only one element and the arrow is not displayed correctly (see below)
* fix closing icon for use case modal - isn't visible for small screens (works for initiatives) - see below
* wider top margin between modal border and text

image

image

**Comment by JoannaNitek** [comment]
Zrzut ekranu z 2023-02-24 16-33-28
Zrzut ekranu z 2023-02-24 16-32-59

**Comment by malwinq** [comment]
Looks better, thanks! For a matter of time, I made few small fixes:

- did we want to remove "Used by: You?" section from the main page? - I reverted this change for now
- Contributors section in Team tab is broken - wasn't centered for me, I added justify-content: center; and display: inline-block; instead of flex
- removed some commented blocks

**Pull request merged**